eclass/pam.eclass

```diff
@@ -171,26 +171,25 @@ pamd_mimic_system() {
 pamd_mimic() {
 	[[ $# -lt 3 ]] && die "pamd_mimic requires at least three arguments"
 
-	if has pam ${IUSE} && ! use pam; then
+	if in_iuse pam && ! use pam; then
 		return 0;
 	fi
 
 	dodir /etc/pam.d
-	pamdfile=${D}/etc/pam.d/$2
+	local pamdfile="${ED}/etc/pam.d/$2"
 	echo -e "# File autogenerated by pamd_mimic in pam eclass\n\n" >> \
-		$pamdfile
+		"${pamdfile}" || die
```

Comment:
The `>>` at the end of the line looks strange, maybe move it to the following line?